package/MDAnalysis/analysis/hydrogenbonds/hbond_analysis.py

```diff
@@ -442,9 +442,8 @@ def _get_dh_pairs(self):
     def _prepare(self):
 
         self.hbonds = [[], [], [], [], [], []]
-        self.frames = np.arange(self.start, self.stop, self.step)
-        self.timesteps = (self.frames * self.u.trajectory.dt) + self.u.trajectory[0].time
-
+        self.timesteps = self.times
```

Comment:
Nice catch 👍 

Rather than creating a case where we have two attributes that do the same thing, I would suggest here just deperecating `self.timesteps` in favour for `self.times`?